### DISCUSSION TITLE
[NHẬN TASK]

### DISCUSSION THREAD

**Discussion by nguyenxuanquang6789** [Runway-Club/Runway-CV-Builder#96] [Task Delivery]
Tôi muốn nhận task: Căn chỉnh lại các UI

**Comment by nguyentantrung2000** [comment]
Thông tin task:
Chỉnh sửa lại các ui chưa hoàn thiện xong
Yêu cầu:
Phải phù hợp với người dùng 
Deadline: 21/8/2021 10:59AM

**Reply by nguyenxuanquang6789** [reply]
ok
